css_tasks/task6/main.css

```diff
@@ -0,0 +1,453 @@
+:root {
+  --logoFont: "Inconsolata", monospace;
+  --mainFont: "Playfair Display", serif;
+  --titleFont: "Ubuntu", sans-serif;
+  --logoColor: #000;
+  --mainColor: #626262;
+  --titleColor: #b4ad9e;
+  --backgroundColor: #f0f0f0;
+  --mainFontSize: 0.875rem;
+  --biggerFontSize: 1.875rem;
+  --mdHeaderFontSize: 1.125rem;
+}
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body {
+  margin: 0 auto;
+}
+.container {
+  width: 80%;
+  max-width: 66.5rem;
+  margin: 0 auto;
+}
+/************************* 
+*                        *
+*        HEADER          *
+*                        *
+*************************/
+.header {
+  display: flex;
+  margin-top: 4.25rem;
+}
+.header__logo {
+  height: 1.4375rem;
+  margin-right: 23rem;
+  color: var(--logoColor);
+  font-family: var(--logoFont);
+  font-size: var(--biggerFontSize);
+  letter-spacing: 6px;
+  line-height: 0.8;
+  text-transform: uppercase;
+}
+.header__navigation {
+  width: 35.625rem;
+}
+.header__navigation-list {
+  display: flex;
+  justify-content: space-between;
```

Comment:
It's close to margins instead of `space-between`.
More appropriate is `space-between` logo and navigation
in this case `.header__navigation` would not have fixed width

Reply:
Done